src/main/java/com/gamingmesh/jobs/PlayerManager.java

```diff
@@ -1024,9 +1024,11 @@ public Boost getFinalBonus(JobsPlayer player, Job prog, Entity ent, LivingEntity
 	if (Jobs.getGCManager().useDynamicPayment)
 	    boost.add(BoostOf.Dynamic, new BoostMultiplier().add(prog.getBonus()));
 //	boost.add(BoostOf.Item, Jobs.getPlayerManager().getItemBoost(player.getPlayer(), prog));
-	boost.add(BoostOf.Item, getItemBoostNBT(player.getPlayer(), prog));
-	boost.add(BoostOf.Area, new BoostMultiplier().add(Jobs.getRestrictedAreaManager().getRestrictedMultiplier(player.getPlayer())));
-
+        if (player.isOnline()) {

```

Comment:
This is not the right place to check if a player is available. Place this check at the start of `getFinalBonus` after the null check.